part-ii-domain-logic/command-model.md

```diff
@@ -116,6 +116,40 @@ It is possible to `apply()` new events inside an Event Sourcing Handler method.
 
 You can also use the static `AggregateLifecycle.isLive()` method to check whether the aggregate is 'live'. Basically, an aggregate is considered live if it has finished replaying historic events. While replaying these events, isLive\(\) will return false. Using this `isLive()` method, you can perform activity that should only be done when handling newly generated events.
 
+## Spawning a new Aggregate
+
+New Aggregate can be created by issuing a creation command, can be created from a Saga (again by issuing a creation command), but in some cases it can be created from an Aggregate. Prior to version 3.3, this had to be orchestrated via Saga. Version 3.3 introduces a functionality to create a new Aggregate from an Aggregate. `AggregateLifecycle` introduces static method `createNew`. 
```

Comment:
The creation of one aggregate based on an event happening is not only doable from a Saga.
You could just as well have a Event Processor which will listen to the event and as a response to that publish a command creating another aggregate.
I'd thus suggest editing this section a little to not only reference Sagas, but talk about Event Handling Components like Sagas or regular Event Processors.

Reply:
Good suggestion!